backend/src/controllers/profileController.js

```diff
@@ -0,0 +1,30 @@
+export default class ProfileController {
+    static profiles = []; // TODO: replace with database
+
+    static getProfiles(req, res) {
+        res.json(ProfileController.profiles);
+    }
+
+    static getProfile(req, res) {
+        const userId = req.params.id;
+        const user = ProfileController.profiles.find(user => user.id === userId);
+        if (user) {
+            res.json(user);
+        } else {
+            res.status(404).json({ message: `User with id = ${userId} not found :(` });
+        }
+    }
+
+    static createProfile(req, res) {
+        const { name, email, hobby } = req.body;
+        console.log(name+email+hobby+"!!!!!")
```

Comment:
Plus sign can have a lot of side effects depending on the actual data, so it's better to replace it with the template string